person.rb

```diff
@@ -1,15 +1,16 @@
 require './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Kindly make the `rentals` array variable have a getter and setter method since we would need to update a `Book` or `Person` rental anytime that `Book` is rented or a `Person` wants to rent a book. 😄